.github/workflows/upstream-sync.yml

```
@@ -0,0 +1,101 @@
+          if [ "$BEHIND" -eq 0 ]; then
+            echo "Up to date with upstream. Skipping sync."
+          else
+            echo "Behind upstream by $BEHIND commits."
+          fi
+
+      - name: Run upstream sync
+        if: steps.check.outputs.behind != '0'
+        env:
+          CLAUDE_CODE_OAUTH_TOKEN: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+          GH_TOKEN: ${{ secrets.GH_PAT }}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:7:32: Double quote to prevent globbing and word splitting

(shellcheck)